.github/deploy_test_preview.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+GITHUB_PAGES_DIR=$TRAVIS_BUILD_DIR/tests/gh-pages
+
+MONOLITH_DOWNLOAD_URL=https://github.com/Y2Z/monolith/releases/download/v2.3.1/monolith-gnu-linux-x86_64
```

Comment:
I'm pretty sure we don't need to use monolith here....

Reply:
Let's move the discussion to #58